ceph_medic/checks/common.py

```diff
@@ -70,3 +70,19 @@ def check_cluster_fsid(host, data):
 
     if mismatched_hosts:
         return code, msg % (current_fsid, ','.join(mismatched_hosts))
+
+
+def check_ceph_version_parity(host, data):
+    code = 'ECOM6'
+    msg = 'Ceph version "%s" is different than host(s): %s'
```

Comment:
Should this message mention that this is the installed ceph version and not the version the daemons report? Just so this is clearly different than #65